FailureLink.py

```diff
@@ -56,102 +56,121 @@
 # If the NZB download of a Movie or TV Show fails, request an alternate
 # NZB-file of the same release and add it to queue. If disabled the indexer
 # site is still informed about the failure but no other nzb-file is queued.
-#DownloadAnotherRelease=no
+# DownloadAnotherRelease=no
```

Comment:
I'm running 21.0 stable and the space between the "#" and "DownloadAnotherRelease=no" breaks the configuration screen for this extension script. The same comment applies to all of the config properties.

Reply:
Hmmm you are right. Will fix.